ganga/GangaTutorial/__init__.py

```diff
@@ -2,8 +2,8 @@
 
 def loadPlugins( config = {} ):
 
-    import Lib.PrimeFactorizer
-    import Lib.PrimeFactorizerSplitter
-    import Lib.PrimeTableDataset
+    import GangaTutorial.Lib.PrimeFactorizer
+    import GangaTutorial.Lib.PrimeFactorizerSplitter
```

Comment:
I think this would better be
```
    import .Lib.PrimeFactorizer
```
and the same for all the other lines.

Reply:
@egede 
the line
import .Lib.PrimeFactorizer
is showing the following error
![Screenshot from 2020-02-13 13-50-02](https://user-images.githubusercontent.com/40742522/74414689-eb082900-4e67-11ea-84b8-7a7495e30043.png)

Reply:
Sorry, look at https://github.com/ganga-devs/ganga/blob/e5b5da6f449f6c22cf6b7029896468152e738f5c/ganga/GangaDirac/__init__.py#L111 for the syntax.